tests/datasource/test_new_datasource_with_runtime_data_connector_sql_execution_engine.py

```diff
@@ -194,7 +199,7 @@ def test_batch_identifiers_and_batch_identifiers_success_all_keys_present_with_q
 ):
     # interacting with the database using query
     test_query: str = "SELECT * FROM table_full__I;"
-    batch_identifiers = {
+    batch_identifiers: dict = {
```

Comment:
@Shinnnyshinshin I guess the type hint here is `Dict[str, Union[str, int]]`.  I feel that we should try to be specific (within reason, if course).  Thanks!